language/proto/kinds.go

```diff
@@ -20,7 +20,11 @@ import "github.com/bazelbuild/bazel-gazelle/rule"
 var protoKinds = map[string]rule.KindInfo{
 	"proto_library": {
 		NonEmptyAttrs:  map[string]bool{"srcs": true},
-		MergeableAttrs: map[string]bool{"srcs": true},
+		MergeableAttrs: map[string]bool{
+			"srcs": 						 		true,
```

Comment:
Something weird happened with the whitespace. Please run `gofmt`.

Also, I'm not entirely sure if these should be mergeable. That means Gazelle will override user values without `# keep` comments. It will delete existing attributes if the directives aren't set.

Reply:
The downside I see to making these attributes mergeable is repos that are already using the attributes will either have to start using the directives, or add a lot of `# keep` comments. The benefit of the attributes being mergeable is it makes it easier to mass edit them.